sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,35 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId, baseURL) {
+
+    let token = this._api._token
+
+    let Authorization = null
+    if (typeof token === 'string') {
+      Authorization = `Bearer ${token}`
+    }
+    else if (typeof token === 'function') {
+      const tkn = (await token()).access_token
+      Authorization = `Bearer ${tkn}`
+    }
```

Comment:
I'm a bit swayed with duplicating all this logic from the `Http` class here. If this is a single occurrence that we need to do custom HTTP requests, I think I'm fine. However, we can anticipate more scenarios like these, we should rather look into sharing this logic to avoid code duplication.

Reply:
in the future this should be a documented API endpoint